asyncapi-circe/src/main/scala/sttp/apispec/asyncapi/circe/package.scala

```diff
@@ -18,7 +18,13 @@ package circe {
     // note: these are strict val-s, order matters!
     override val openApi30: Boolean = true
 
-    implicit val encoderOAuthFlow: Encoder[OAuthFlow] = deriveEncoder[OAuthFlow].mapJsonObject(expandExtensions)
+    implicit val encoderOAuthFlow: Encoder[OAuthFlow] = {
+      import scala.collection.immutable.ListMap
```

Comment:
it would be good to add a comment why this is treated differently than the default, just as here: https://github.com/softwaremill/sttp-apispec/blob/d80b0837c7fb92e7dea7cc4e92147efaa5e647f3/openapi-circe/src/main/scala/sttp/apispec/openapi/internal/InternalSttpOpenAPICirceEncoders.scala#L52

Reply:
already fixed